alien4cloud-yorc-plugin/src/main/java/org/ystia/yorc/alien4cloud/plugin/YorcPaaSProvider.java

```diff
@@ -146,6 +148,21 @@ public YorcPaaSProvider() {
         taskManager = new TaskManager(3, 120, 3600);
     }
 
+    public void stopLogsAndEvents() {
+        eventListenerTask.valid = false;
```

Comment:
This is not really "java-style" to set attributes directly.
I would use a setter or a `stop()` method that manages `valid` and make it private

Reply:
Ok done !